checkpoint.go

```diff
@@ -69,7 +69,11 @@ checkpointed.`,
 
 		if !(options.LeaveRunning || options.PreDump) {
 			// destroy container unless we tell CRIU to keep it
-			defer destroy(container)
+			defer func() {
+				if err := killContainer(container); err != nil {
```

Comment:
I don't understand this change. What destroy does is removing the container. What `killContainer` does is sending SIGKILL to the container. Replacing one with the other does not make sense.

I don't understand how it fixes https://github.com/opencontainers/runc/issues/3577 either.

Reply:
@gosoon ^^^^